src/store-update.android.ts

```diff
@@ -13,19 +13,30 @@ app.on(app.resumeEvent, () => {
   StoreUpdate.checkForUpdate()
 })
 
-export class StoreUpdate extends StoreUpdateCommon {
-  private static _checkHasTimeouted = false
-  private static _timeoutChecker
-  private static _looperChecker
+export class StoreUpdate {
+  private static _storeUpdateCommon
+  private static _instantiated = false
 
   /*
-   *  Public
-   */
+  *  Public
+  */
+
+  static init(config: IStoreUpdateConfig) {
+    if (StoreUpdate._instantiated) throw new Error('NS Store Update already configured')
```

Comment:
Why adding/checking a `_instantiated` var? As `_storeUpdateCommon` is a singleton, checking that it is defined should be enough